public/app/features/plugins/admin/components/InstallControls/InstallControlsButton.tsx

```diff
@@ -114,6 +114,11 @@ export function InstallControlsButton({
   };
 
   if (pluginStatus === PluginStatus.UNINSTALL) {
+    const disableUninstall =
+      config.pluginAdminExternalManageEnabled && configCore.featureToggles.managedPluginsInstall
+        ? plugin.isUninstalling
+        : isUninstalling;
```

Comment:
I think it would worth updating the tests as well to cover this scenario in `InstallControlsButton.test.tsx`.